tensordict/_td.py

```diff
@@ -168,6 +168,10 @@ class TensorDict(TensorDictBase):
             tensordict. If provided, its length must match the one of the
             ``batch_size``. Defaults to ``None`` (no dimension name, or ``None``
             for every dimension).
+        non_blocking (bool, optional): if ``True`` and a device is passed, the tensordict
+            is delivered without synchronization. If ``False``, a global synchronization
+            will be made before passing the tensordict, making sure that all the stored
```

Comment:
```suggestion
            will be made before delivering the tensordict, making sure that all the stored
```
To be consistent with with the sentence above